src/pages/workspaces/workspace/workflows-cbas/SubmissionConfig.js

```diff
@@ -0,0 +1,15 @@
+import { h2 } from 'react-hyperscript-helpers';
+import { wrapWorkflowsPage } from 'src/pages/workspaces/workspace/workflows-cbas/WorkflowsContainer';
+
+export const SubmissionConfig = wrapWorkflowsPage({ name: 'SubmissionConfig' })((props, _ref) => {
+  return h2(['Submission Config page']);
+});
+
+export const navPaths = [
+  {
+    name: 'workspace-workflows-cbas-submission-config',
+    path: '/workspaces/:namespace/:name/workflows-cbas/submission-config/:methodId',
```

Comment:
Similar comment about `workflows-cbas` - I think it might be better to keep this generic by calling `workflows-azure` or `azure-workflows` so that its not service specific and Cromwell related pages like Workflow details can also be put under the same path.